### DISCUSSION TITLE
Is Multiuser MIMO OFDM Communication Supported for the CDL Channel Model in Sionna?

### DISCUSSION THREAD

**Discussion by HomerosTheII** [NVlabs/sionna#297] [Q&A] [answered]
I am working on multiuser systems and doing simulations on the channel models UMi, UMa, and RMa. I wish to do simulations with CDL as well, however I am not sure if it is possible since CDL does not have a topology. Would creating a proper multiuser RX-TX association matrix be enough to switch to multiuser systems from single user or is there something else I should do?

**Comment by faycalaa** [comment, marked as answer]
Hi,

CDL model correspond to fixed power delay profiles (see [A] section 7.7.1), and therefore only works for systems with a single transmitter and a single receiver. The transmitter and receiver can however be equipped with multiple antennas.

Hope this helps.

[A] 3GPP TR 38.901, “Study on channel model for frequencies from 0.5 to 100 GHz”

**Reply by HomerosTheII** [reply]
I see. Well that's a bummer.

Thank you for your reply!
